src/main/java/uk/gov/hmcts/ccd/config/JacksonObjectMapperConfig.java

```diff
@@ -19,10 +22,10 @@
     @Primary
     @Bean(name = "DefaultObjectMapper")
     public ObjectMapper defaultObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
-        mapper.registerModules(new JavaTimeModule());
-        mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
-        return mapper;
+        return new ObjectMapper()
+            .registerModule(new Jdk8Module())
```

Comment:
this is the default mapper. Shall we ask some regression testing around this change?

Reply:
tbh not much change there - just added to support ParameterNamesModule. Yes, all this work needs regression testing.